fedoralovespython.py

```diff
@@ -2,34 +2,81 @@
 
 import jinja2
 import yaml
-from flask import Flask, render_template
+from flask import Flask, Response, render_template, jsonify, g
 from markdown import markdown
-
+import ansicolor
 
 app = Flask('fedoralovespythonorg')
 app.config['TEMPLATES_AUTO_RELOAD'] = True
 
 
+def read_points(hide_upcoming=True):
+    points = read_yaml('points.yml')
+    if hide_upcoming:
+        points = [p for p in points if p.get('ready', True)]
+    return points
+
+
 @app.route('/')
 def index():
-    points = read_yaml('points.yml')
-    points = [p for p in points if p.get('ready', True)]
+    points = read_points()
     return render_template('index.html', points=points)
 
 
 @app.route('/__future__/')
 def future():
-    points = read_yaml('points.yml')
+    points = read_points(hide_upcoming=False)
     return render_template('index.html', points=points)
 
 
+@app.route('/index.txt')
+def plaintext():
+    points = read_points()
+    g.colors = False
+    return Response(
+        render_template('index.txt', points=points),
+        mimetype='text/plain'
+    )
+
+
+@app.route('/index.term')
+def terminal():
+    points = read_points()
+    g.colors = True
+    return Response(
+        render_template('index.txt', points=points),
+        mimetype='application/octet-stream'
+    )
+
+
+@app.route('/index.json')
+def json():
+    points = read_points()
+    return jsonify(points)
```

Comment:
Using `jsonify` directly to points is easy, but it results in:

    "logo": "science"

And that is not quite useful. Maybe we could change the image slug into `url_for('static', filename='img/' + point.logo + '.svg')` in `read_points` and return the URL in JSON and use it directly in the `index.html` template as well?